src/sources/file.rs

```diff
@@ -80,26 +82,26 @@ pub struct FileConfig {
 
 #[derive(Deserialize, Serialize, Clone, Debug, PartialEq)]
 #[serde(tag = "strategy", rename_all = "snake_case")]
-pub enum FingerprintingConfig {
+pub enum FingerprintConfig {
     Checksum {
-        fingerprint_bytes: usize,
```

Comment:
This is a breaking change, should we add serde alias here too?

Reply:
Yes, we should. Would you mind opening a PR for that? I don't think James or myself know how to do that.

Reply:
Submitted!